### PR TITLE
[plugin][actionlog][joomla] - Notice when disabled

### PR DESCRIPTION
**Pull request by alikon**
Pull Request for Issue https://github.com/joomla-projects/privacy-framework/pull/236 .

### Summary of Changes
redo of https://github.com/joomla-projects/privacy-framework/pull/236

in the right branch

### CODE REVIEW

File: plugins/system/actionlogs/actionlogs.php [hunk 1]
```diff
@@ -119,6 +119,13 @@ public function onContentPrepareForm($form, $data)
 		}
 
 		JForm::addFormPath(dirname(__FILE__) . '/forms');
+
+		if (!JPluginHelper::isEnabled('actionlog', 'joomla'))
+		{
+			$form->loadFile('information', false);
+			return true;
```
Comment (Quy) [inline plugins/system/actionlogs/actionlogs.php]: Add blank line above.

**Review by brianteeman — APPROVED** [review]
language strings approved

File: administrator/components/com_actionlogs/models/fields/plugininfo.php [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+		try
+		{
+			$result = (int) $db->loadResult();
+		}
+		catch (RuntimeException $e)
+		{
+			JError::raiseWarning(500, $e->getMessage());
+		}
+
```
Comment (Quy) [inline administrator/components/com_actionlogs/models/fields/plugininfo.php]: Remove tab

**Review by mbabker — APPROVED** [review]

File: plugins/system/actionlogs/actionlogs.php (hunk 1 above)
Comment (alikon) [reply plugins/system/actionlogs/actionlogs.php]: done

File: administrator/components/com_actionlogs/models/fields/plugininfo.php (hunk 2 above)
Comment (alikon) [reply administrator/components/com_actionlogs/models/fields/plugininfo.php]: done

File: plugins/system/actionlogs/forms/information.xml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<form>
+	<fields name="params">
+		<fieldset name="information" label="PLG_SYSTEM_ACTIONLOGS_OPTIONS" addfieldpath="/administrator/components/com_actionlogs/models/fields">
+			<field
+				name="Information"
+				type="plugininfo"
+			>
+			</field>
```
Comment (Quy) [inline plugins/system/actionlogs/forms/information.xml]: Remove `</field>`

File: administrator/language/en-GB/en-GB.plg_system_actionlogs.ini [hunk 4]
```diff
@@ -6,6 +6,8 @@
 PLG_SYSTEM_ACTIONLOGS="System - User Actions Log"
 PLG_SYSTEM_ACTIONLOGS_EXTENSIONS_NOTIFICATIONS="Select events to be notified for"
 PLG_SYSTEM_ACTIONLOGS_EXTENSIONS_NOTIFICATIONS_DESC="Select events to be sent to your email as notifications"
+PLG_SYSTEM_ACTIONLOGS_JOOMLA_ACTIONLOG_DISABLED="Action Log - Joomla"
+PLG_SYSTEM_ACTIONLOGS_JOOMLA_ACTIONLOG_DISABLED_REDIRECT="The \"%s\" plugin is disabled."
```
Comment (Quy) [inline administrator/language/en-GB/en-GB.plg_system_actionlogs.ini]: Make it same as these plugins without double quotes and capitalize/link `plugin`:
```
The Smart Search Content Plugin is disabled.
The Redirect System Plugin is disabled.
```

File: plugins/system/actionlogs/forms/information.xml (hunk 3 above)
Comment (alikon) [reply plugins/system/actionlogs/forms/information.xml]: why ?
removing `</field> ` make the xml file invalid

File: administrator/language/en-GB/en-GB.plg_system_actionlogs.ini (hunk 4 above)
Comment (alikon) [reply administrator/language/en-GB/en-GB.plg_system_actionlogs.ini]: it's a bit different from that two

File: plugins/system/actionlogs/forms/information.xml (hunk 3 above)
Comment (mbabker) [reply plugins/system/actionlogs/forms/information.xml]: Should be a self closing tag if there are no child elements.

Comment (Quy) [reply plugins/system/actionlogs/forms/information.xml]: Change to. Also add `label` attribute so `Information` can be localized?
```
			<field
				name="Information"
				type="plugininfo"
			/>
```

Comment (alikon) [reply plugins/system/actionlogs/forms/information.xml]: done + done

File: administrator/language/en-GB/en-GB.plg_system_actionlogs.ini (hunk 4 above)
Comment (alikon) [reply administrator/language/en-GB/en-GB.plg_system_actionlogs.ini]: done

File: administrator/language/en-GB/en-GB.plg_system_actionlogs.ini [hunk 5]
```diff
@@ -6,6 +6,10 @@
 PLG_SYSTEM_ACTIONLOGS="System - User Actions Log"
 PLG_SYSTEM_ACTIONLOGS_EXTENSIONS_NOTIFICATIONS="Select events to be notified for"
 PLG_SYSTEM_ACTIONLOGS_EXTENSIONS_NOTIFICATIONS_DESC="Select events to be sent to your email as notifications"
+PLG_SYSTEM_ACTIONLOGS_JOOMLA_ACTIONLOG_DISABLED="Action Log - Joomla"
+PLG_SYSTEM_ACTIONLOGS_JOOMLA_ACTIONLOG_DISABLED_REDIRECT="The %s Plugin is disabled."
+PLG_SYSTEM_ACTIONLOGS_INFO_DESC="The Action Log - Joomla plugin is disabled"
```
Comment (Quy) [inline administrator/language/en-GB/en-GB.plg_system_actionlogs.ini]: Move lines 11-12 above line 9

Comment (alikon) [reply administrator/language/en-GB/en-GB.plg_system_actionlogs.ini]: sorted

**Comment by Quy** [comment]
Here is the error message when clicking the link on the frontend: `404 Component not found. `

```
http://localhost/joomla-cms-staging/index.php/component/plugins/?task=plugin.edit&extension_id=484
```

File: administrator/language/en-GB/en-GB.plg_system_actionlogs.ini [hunk 6]
```diff
@@ -6,6 +6,10 @@
 PLG_SYSTEM_ACTIONLOGS="System - User Actions Log"
 PLG_SYSTEM_ACTIONLOGS_EXTENSIONS_NOTIFICATIONS="Select events to be notified for"
 PLG_SYSTEM_ACTIONLOGS_EXTENSIONS_NOTIFICATIONS_DESC="Select events to be sent to your email as notifications"
+PLG_SYSTEM_ACTIONLOGS_INFO_DESC="The Action Log - Joomla plugin is disabled"
+PLG_SYSTEM_ACTIONLOGS_INFO_LABEL="Information"
+PLG_SYSTEM_ACTIONLOGS_JOOMLA_ACTIONLOG_DISABLED="Action Log - Joomla"
+PLG_SYSTEM_ACTIONLOGS_JOOMLA_ACTIONLOG_DISABLED_REDIRECT="The %s Plugin is disabled."
```
Comment (infograf768) [inline administrator/language/en-GB/en-GB.plg_system_actionlogs.ini]: lowercase for plugin?

**Comment by alikon** [comment]
@Quy good catch should be ok now with https://github.com/joomla/joomla-cms/pull/21898/commits/98c92d8b51be92111dec9ddd98dfe2038aefa5d2

File: administrator/language/en-GB/en-GB.plg_system_actionlogs.ini (hunk 6 above)
Comment (alikon) [reply administrator/language/en-GB/en-GB.plg_system_actionlogs.ini]: upper vs lower ?
https://github.com/joomla/joomla-cms/pull/21898#pullrequestreview-154039383
just tell me what is the standard please

Comment (infograf768) [reply administrator/language/en-GB/en-GB.plg_system_actionlogs.ini]: lower when in a phrase I guess

Comment (infograf768) [reply administrator/language/en-GB/en-GB.plg_system_actionlogs.ini]: But I can live with it as long as no one is changing this later on...

Comment (brianteeman) [reply administrator/language/en-GB/en-GB.plg_system_actionlogs.ini]: yes please change it to lower case just as it is in the _DESC

Comment (alikon) [reply administrator/language/en-GB/en-GB.plg_system_actionlogs.ini]: ok the let's go with lower case now

**Review by brianteeman — APPROVED** [review]
approve language changes

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 30f0d00200c252b742885310afea6e81e18ff723<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/21898">issues.joomla.org/tracker/joomla-cms/21898</a>.</sub>

**Comment by viocassel** [comment]
I have tested this item :white_check_mark: successfully on 30f0d00200c252b742885310afea6e81e18ff723<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/21898">issues.joomla.org/tracker/joomla-cms/21898</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/21898">issues.joomla.org/tracker/joomla-cms/21898</a>.</sub>

**Pull request merged**